WalletWasabi/WebClients/Coinstamp/CoinstampExchangeRateProvider.cs

```diff
@@ -0,0 +1,30 @@
+﻿#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using WalletWasabi.Backend.Models;
+using WalletWasabi.Interfaces;
+
+namespace WalletWasabi.WebClients.Coinstamp
+{
+	public partial class CoinstampExchangeRateProvider : IExchangeRateProvider
+	{
+		public async Task<List<ExchangeRate>> GetExchangeRateAsync()
```

Comment:
It would be nice to document it a bit:

```suggestion
		/// <summary>
		/// Gets exchange rate from Bitstamp.net.
		/// </summary>
		/// <remarks>
		/// This method may throw various exceptions for various reasons.
		/// <para>JSON response format is: <c>{"high": "11457.95", "last": "11375.68", "timestamp": "1596233998", "bid": "11375.68", "vwap": "11232.95", "volume": "10352.38275319", "low": "10973.80", "ask": "11377.06", "open": "11110.84"}</c></para>
		/// </remarks>
		/// <returns>Top of the book bid price.</returns>
		public async Task<List<ExchangeRate>> GetExchangeRateAsync()
```

Reply:
These providers never change and their complexity is zero, there is no reason to document them in this way. In fact, if one day one of these start to fail, it will be because the provider changed the endpoint or the response (the json) and in that case the developer cannot trust in the documentation and will need to invest 5 or 10 seconds as you and I did.

Reply:
What is the downside of commenting them like that?

Complexity of the code is low but you say how others are supposed to use it correctly. When the API changes, you actually know what has changed too. So I find it useful. And it took like a minute of effort so basically nothing.

Reply:
It seems I marked this as resolved but I don't remember me doing so. Anyway, I find a downside in comments, especially iif i think they are unnecessary as in this case. 
I would appreciate if we could move on and break this stalemate because this is one of the "very important" PR listed in project 4 and it makes no sense to block it for a comment.

Reply:
I don't mean to block you because of this of course. It's a suggestion. I think it makes perfect sense but people don't agree with me on this too much so ... I'm used to this :))